src/status_im/hardwallet/common.cljs

```diff
@@ -166,26 +174,36 @@
       :on-connect    ::on-card-connected
       :on-disconnect ::on-card-disconnected})))
 
-(fx/defn show-pair-sheet
-  [cofx {:keys [on-cancel]
-         :or   {on-cancel [::cancel-sheet-confirm]}}]
-  (log/debug "[hardwallet] show-pair-sheet")
-  (let [connected? (get-in cofx [:db :hardwallet :card-connected?])]
-    (fx/merge cofx
-              {:dismiss-keyboard true}
-              (bottom-sheet/show-bottom-sheet
-               {:view {:show-handle?      false
-                       :backdrop-dismiss? false
-                       :disable-drag?     true
-                       :content           (keycard-sheet-content on-cancel
-                                                                 connected?)}}))))
-
-(fx/defn hide-pair-sheet
+(fx/defn show-connection-sheet
+  [{:keys [db] :as cofx} {:keys [on-card-connected on-card-read handler]
+                          {:keys [on-cancel]
+                           :or   {on-cancel [::cancel-sheet-confirm]}}
```

Comment:
Maybe this could be on the same map level as other options?

Reply:
of course we can, if there is any particular reason to do so